src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/repository/DisputeRepository.java

```diff
@@ -6,19 +6,23 @@
 import java.util.Optional;
 
 import ca.bc.gov.open.jag.tco.oracledataapi.model.Dispute;
+import ca.bc.gov.open.jag.tco.oracledataapi.model.DisputeListItem;
 import ca.bc.gov.open.jag.tco.oracledataapi.model.DisputeResult;
 import ca.bc.gov.open.jag.tco.oracledataapi.model.DisputeStatus;
 
 public interface DisputeRepository {
 
-	/** Fetch all records older than the given date. */
-	public List<Dispute> findByCreatedTsBefore(Date olderThan);
+	/** Fetch all records newer than the given date. */
+	public List<DisputeListItem> findByCreatedTsAfter(Date newerThan);
```

Comment:
Yeah, let's keep the code as is for now and we'll have a cleanup task apart from this PR to remove dead code.

Reply:
Yes, this changes the intended use only for the H2 database implementation which we don't even use anymore. It actually matches what the Oracle database query does with that change. 
That is my observation as well that it its not even used in Staff API and always passed in null.
So, yeah I think it would be good to clear out the code that's not necessary at some point after confirming with front-end and business.